src/test/java/unit/GitRepoReferenced.java

```diff
@@ -201,52 +201,55 @@ void updateIndividualProperties() throws IOException {
     // update just the name
     // `terra resources update git-repo --name=$name --new-name=$newName`
     String newName = "updateIndividualProperties_NEW";
-    UFGitRepo updateGitRepo =
+    UFGitRepo updatedGitRepo =
         TestCommand.runAndParseCommandExpectSuccess(
             UFGitRepo.class,
             "resource",
             "update",
             "git-repo",
             "--name=" + name,
             "--new-name=" + newName);
-    assertEquals(newName, updateGitRepo.name);
-    assertEquals(description, updateGitRepo.description);
+    assertEquals(newName, updatedGitRepo.name);
+    assertEquals(description, updatedGitRepo.description);
 
     // `terra resources describe --name=$newName`
-    UFGitRepo describeGitRepo =
+    UFGitRepo describedGitRepo =
         TestCommand.runAndParseCommandExpectSuccess(
             UFGitRepo.class, "resource", "describe", "--name=" + newName);
-    assertEquals(description, describeGitRepo.description);
+    assertEquals(description, describedGitRepo.description);
 
     // update just the description
-    // `terra resources update git-repo --name=$newName --description=$newDescription`
+    // `terra resources update git-repo --name=$newName --description=$newDescription
+    // --new-cloning=COPY_REFERENCE`
     String newDescription = "updateDescription_NEW";
-    updateGitRepo =
+    updatedGitRepo =
         TestCommand.runAndParseCommandExpectSuccess(
             UFGitRepo.class,
             "resource",
             "update",
             "git-repo",
             "--name=" + newName,
-            "--description=" + newDescription);
-    assertEquals(newName, updateGitRepo.name);
-    assertEquals(newDescription, updateGitRepo.description);
+            "--description=" + newDescription,
+            "--new-cloning=" + CloningInstructionsEnum.DEFINITION);
+    assertEquals(newName, updatedGitRepo.name);
+    assertEquals(newDescription, updatedGitRepo.description);
 
     // `terra resources describe --name=$newName`
-    describeGitRepo =
+    describedGitRepo =
         TestCommand.runAndParseCommandExpectSuccess(
             UFGitRepo.class, "resource", "describe", "--name=" + newName);
-    assertEquals(newDescription, describeGitRepo.description);
+    assertEquals(newDescription, describedGitRepo.description);
+    assertEquals(CloningInstructionsEnum.REFERENCE, describedGitRepo.cloningInstructions);
```

Comment:
here again I thought the cloning should be `DEFINITION` based on the update command.